### ISSUE TITLE
Version 0.16.0 is generating None: Optional as a property in Pydantic models.

### ISSUE THREAD

**Issue by jcgentr** [koxudaxi/datamodel-code-generator#1021]
**Describe the bug**
Version 0.16.0 is generating `None: Optional` as a property in Pydantic models.

**To Reproduce**

Use datamodel-codegen version 0.15.0
Then use datamodel-codegen version 0.16.0 

Used commandline:
```
$ datamodel-codegen --input api/machina-common-swagger.yaml --output models.py --target-python-version 3.9
```

**Expected behavior**
I expect my generated models file not to contain Python errors.

**Version:**
 - OS: macOS
 - Python version: 3.10.9
 - datamodel-code-generator version: 0.15.0
 
<img width="777" alt="Screen Shot 2023-01-17 at 5 47 11 PM" src="https://user-images.githubusercontent.com/20710009/213036967-14da8da0-1292-49b5-931f-5e8ad56b9936.png">

 - OS: macOS
 - Python version: 3.10.9
 - datamodel-code-generator version: 0.16.0
 
<img width="768" alt="Screen Shot 2023-01-17 at 5 42 43 PM" src="https://user-images.githubusercontent.com/20710009/213036455-5e33053e-7a81-448e-864d-552a01ca1f00.png">

**Comment by koxudaxi** [comment]
@jcgentr 
Thank you for creating the issue.
Could you please share your schema?

**Comment by koxudaxi** [comment]
@jcgentr   
I have released the new version as `0.16.1`
Thank you very much!!

**Comment by jcgentr** [comment]
That worked for me, thanks!
